src/main/java/liquibase/ext/mongodb/database/MongoConnection.java

```diff
@@ -50,10 +55,13 @@
 @NoArgsConstructor
 public class MongoConnection extends AbstractNoSqlConnection {
 
+    private final Logger log = Scope.getCurrentScope().getLog(getClass());
     public static final int DEFAULT_PORT = 27017;
     public static final String MONGO_PREFIX = MONGODB_PRODUCT_SHORT_NAME + "://";
     public static final String MONGO_DNS_PREFIX = MONGODB_PRODUCT_SHORT_NAME + "+srv://";
 
+    public static final String DEFAULT_CHARSET = "UTF-8";
```

Comment:
There is a `fileEncoding` global setting. That probably shouldn't be used here? But making sure you were remembering it in case you think it should be.